content/basics/templates/outputs/index.md

```diff
@@ -53,9 +52,42 @@ In this Lab, you will:
         Value: !GetAtt WebServerInstance.PublicDnsName
     ```
 
-5. Go to the AWS console and update your stack with a new template.
+1. Update the stack with new template.
+
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace`:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+cd cfn101-workshop/code/workspace
+:::
+1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **MyAmiId** with  the value you have hardcoded in `resources.yaml` file earlier.
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+aws cloudformation update-stack --stack-name cfn-workshop-resources --template-body file://outputs.yaml --parameters ParameterKey="AmiID",ParameterValue="MyAmiId"
+:::
+1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-resources/739fafa0-e4d7-11ed-a000-12d9009553ff"
+:::
+ 1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **UPDATE_COMPLETE**.
+::::
+
+::::tab{id="local" label="Local development"}
+1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** link in a new tab and log in to your AWS account.
+1. Click on the stack name, for example **cfn-workshop-resources**.
+1. In the top right corner click on **Update**.
+1. In **Prepare template**, choose **Template is ready**.
+1. In **Template source**, choose **Upload a template file**.
+1. Click on **Choose file** button and navigate to your workshop directory.
+1. Select the file `outputs.yaml` and click **Next**.
+1. You can leave **Configure stack options** default, click **Next**.
+1. On the **Review <stack_name>** page, scroll down to the bottom and click on **Update stack**.
+1. You can click the **refresh** button a few times until you see in the status **UPDATE_COMPLETE**.
```

Comment:
Could you please update this section to create new stack. This was previously wrongly assuming to update stack where we want to create new one.

Reply:
I will do this.